webapp/src/pages/home/Home.js

```diff
@@ -97,6 +96,10 @@ class Home extends Component {
         }
         return <div></div>
     }
+    
+    renderButton = (definition) => {
```

Comment:
Not sure you really need this function. The `renderEventTable` is done that way because we're applying a fairly complex template multiple times, but in this case I think it's clearer to just have the HTML directly where you want the button to go :)

Reply:
Done. And also removed new_event_definition - just creating the button directly